src/components/Navigationbar.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/void-dom-elements-no-children */
```

Comment:
- [ ] `required`
- [ ] please enable the linters because it defeats the purpose of installing linters